### ISSUE TITLE
Ring panic buttons dont show up in Home

### ISSUE THREAD

**Issue by becket-ports** [dgreif/ring#1420]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe The Bug

I have the show panic buttons option selected in the configs but it still doesn’t show up in home app. I’m on the latest plugin version. 

### Expected behavior

See a switch in home app as an accessory for the ring panic buttons

### Homebridge Ring Config

```shell
{
            "refreshToken": "xxx",
            "unbridgeCameras": true,
            "alarmOnEntryDelay": true,
            "hideLightGroups": true,
            "hideDoorbellSwitch": true,
            "hideCameraLight": true,
            "hideCameraMotionSensor": false,
            "hideCameraSirenSwitch": false,
            "hideInHomeDoorbellSwitch": true,
            "hideAlarmSirenSwitch": false,
            "showPanicButtons": true,
            "_bridge": {
                "username": "xxx",
                "port": xxx
            },
            "platform": "Ring"
        },
```

### OS

Debian bookworm

### Node.js Version

v20.12.2

### NPM Version

v10.5.0

### Homebridge/HOOBs Version

v1.8.1

### Homebridge Ring Plugin Version

v12.1.0

### Operating System

Raspbian

**Comment by dgreif** [comment]
@becket-ports I looked into this a bit and was not able to replicate the panic buttons not showing up. You should have an accessory _somewhere_ named "Panic Buttons". Be sure to check every room in the Home app as sometimes accessories are added in weird places.

That said, I did notice that the switches within the accessory _both_ show up as "Panic Buttons", rather than "Fire Alarm" and "Burglar Alarm". I have a possible fix for this issue, which should be included in the next release

**Comment by dgreif** [comment]
I just released v13.0.1 which adds proper names to these switches. Please update and let me know if you are still seeing any issues with panic buttons.
